lib/Event/AdminEvents.php

```diff
@@ -143,6 +143,34 @@ final class AdminEvents
      */
     const OBJECT_LIST_BEFORE_LIST_LOAD = 'pimcore.admin.object.list.beforeListLoad';
 
+    /**
+     * Allows you to modify the object list before it is prepared for export.
+     *
+     * Subject: A controller extending \Pimcore\Bundle\AdminBundle\Controller\AdminController
+     * Arguments:
+     *  - list | the object list
+     *  - context | contains contextual information
+     *
+     * @Event("Pimcore\Event\Model\GenericEvent")
+     *
+     * @var string
+     */
+    const OBJECT_LIST_BEFORE_LIST_PREPARE = 'pimcore.admin.object.list.beforeListPrepare';
+
+    /**
+     * Allows you to modify the object list before it is exported.
+     *
+     * Subject: A controller extending \Pimcore\Bundle\AdminBundle\Controller\AdminController
+     * Arguments:
+     *  - list | the object list
+     *  - context | contains contextual information
+     *
+     * @Event("Pimcore\Event\Model\GenericEvent")
+     *
+     * @var string
+     */
+    const OBJECT_LIST_BEFORE_LIST_EXPORT = 'pimcore.admin.object.list.beforeListExport';
```

Comment:
maybe `'pimcore.admin.object.list.beforeExport'`?

Reply:
I wanted to keep in line with `'pimcore.admin.object.list.beforeListLoad'`

Reply:
It should be clear enough to understand the context. `'pimcore.admin.object.list.beforeListLoad'` does not seem to be related to export.